src/libs/feature-previews.js

```diff
@@ -37,7 +37,7 @@ export const getAvailableFeaturePreviews = async ({ signal } = {}) => {
         () => true
       )
     ),
-    _.sortBy('title')
+    _.sortBy('order')
```

Comment:
Instead of adding an `order` field, could we remove the `sortBy` and let the array in feature-previews-config determine the display order?